src/rest.js

```diff
@@ -35,6 +35,10 @@ export default class RestClient extends Vue {
     }));
   }
 
+  get url() {
+    return new URL(this.apiRoot);
```

Comment:
This actually throws an exception in production, where the default `apiRoot` is "/api/v1". I think we can get rid of this method and let callers do this in the uncommon case where they need to parse the URL (YAGNI).